README.md

```diff
@@ -28,10 +50,18 @@ To create a `libgmock.a` file in a Linux system follow these instructions in an
 5. Collect both the resulting .o files into an archive file:
     `ar -rv libgmock.a gtest-all.o gmock-all.o`
 
-Congratulations! You now have a `libgmock.a` archive file, include it in the `../build/.lib/` directory and you should be good to go, run `qmake` then `make` in the Tests directory to check if you can compile the tests. Use `ls -a` to show all hidden files (including ones starting with `.`)
+Now you must run `qmake` in the `Hermes/src/Tests/` directory, which will
+generate a `Hermes/build/.lib` directory.
+
+Congratulations! You now have a `libgmock.a` archive file, include it in
```

Comment:
The two paragraphs on 53 and 56 seem a little off, as if `qmake` had a step in creating the `libgmock.a` file.